x-pack/plugins/cases/docs/openapi/paths/api@cases@{caseid}@comments@{commentid}.yaml

```
@@ -0,0 +1,160 @@
+post:
+  description: >
+    Adds a comment to a case.
+    You must have all privileges for the **Cases** feature in the **Management**, **Observability**, or **Security** section of the Kibana feature privileges, depending on the owner of the case you're creating.
+  tags:
+    - cases
+    - kibana
+  parameters:
+    - $ref: '../components/headers/kbn_xsrf.yaml'
+    - $ref: '../components/parameters/case_id.yaml'
+  requestBody:
+    content:
+      application/json:
+        schema:
+          type: object
+          properties:
+            alertId: 
+              description: The alert identifier.
+              type: string 
+            comment:
+              description: The new comment.
+              type: string
+            index:
+              description: The alert index.
+              type: string
+            owner:
+              $ref: '../components/schemas/owners.yaml'
+            rule:
+              description: The rule that is associated with the alert.
+              type: object
+              properties:
+                id:
+                  description: The rule identifier.
+                  type: string
+                name: 
+                  description: The rule name.
+                  type: string
+            type: 
+              $ref: '../components/schemas/comment_types.yaml'
+          required:
+            - alertId
+            - comment
+            - index
+            - owner
+            - rule
+            - type
+        examples:
+          createCaseCommentRequest:
+            $ref: '../components/examples/add_comment_request.yaml'
+  responses:
+    '200':
+      description: Indicates a successful call.
+      content:
+         application/json; charset=utf-8:
+          schema:
+            type: object
+            properties:
+              $ref: '../components/schemas/case_response_properties.yaml'
+          examples:
+            createCaseCommentResponse:
+              $ref: '../components/examples/add_comment_response.yaml'
+  servers:
+     - url: https://localhost:5601
+
+delete:
+  description: >
+    Deletes one or all comments from a case.
```

In this case since we're qualifying it with a comment ID, it's only removing a single comment:
```suggestion
    Deletes a comment from a case.
```